### DISCUSSION TITLE
List item detection

### DISCUSSION THREAD

**Discussion by nikitar** [deepdoctection/deepdoctection#358] [General]
What are our options for detecting list items within a list? DDT finds lists very reliably on default settings, but all the children of LayoutType.LIST seem to be LayoutType.WORD annotations. I looked at other models available (print_model_infos) and at the source code, and don't see anything indicating list item detection as a possibility, but wanted to ask just in case.

Thank you for making this!

<img width="864" alt="Screenshot 2024-08-09 at 4 01 24 PM" src="https://github.com/user-attachments/assets/a4151ce4-72a2-405c-9fb0-b6475596320f">
<img width="800" alt="Screenshot 2024-08-09 at 4 03 50 PM" src="https://github.com/user-attachments/assets/3dcedca6-e8f3-41ae-9ba5-9db5e1060938">

**Comment by JaMe76** [comment]
I think you either need to work with rule based approaches (e.g. detecting bullet points) or you need to train a model that detects list items. 
The layout models trained on Publaynet detect the whole list because this is the way they are annotated.

The way lists are annotated on Doclaynet is different, though. Here, lists are actually list items. One idea is to train (or maybe find a model)  on Doclaynet and use the list output to divide your list into items.

I am experimenting with list item detection (esp. for nested lists) as well. My approach is simply to get some samples labelled and see how it works.
